dev-docs/RFCs/v6.0/offscreen-render-rfc.md

```diff
@@ -0,0 +1,100 @@
+# RFC: Offscreen Rendering
+
+* **Authors**: Xiaoji
+* **Date**: March 2018
+* **Status**: Pending Review
+
+Notes:
+- Initial implementation of OffscreenAnimationLoop [#453](https://github.com/uber/luma.gl/pull/453)
+- Example usage of OffscreenAnimationLoop [#454](https://github.com/uber/luma.gl/pull/454)
+
+
+## Motivation
+
+WebGL rendering performance can greatly affect UI responsiveness. The experimental [OffscreenCanvas](https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvas) API in Chrome and Firefox allows us to move WebGL rendering to a worker thread, and free up the main thread to handle DOM interactions. It is highly desirable for heavy front-end applications.
+
+
+## State of the API
+
+OffscreenCanvas is in active development (behind experimental flags) in both FF and [Chrome](https://www.chromestatus.com/feature/5424182347169792). In Chrome, go to `chrome://flags` and enable "Experimental web platform features."
+
+The API still seems limited/unstable:
+- FF only support OffscreenCanvas with WebGL1 context.
+- Chrome's render buffer does not update in some test scenarios.
```

Comment:
Mention the perf issue you found yesterday?

Add Chrome and FF versions and dates as the situation is evolving?